src/Services/privateApiService.js

```diff
@@ -13,4 +13,16 @@ const Get = () => {
     .catch((err) => console.log(err));
 };
 
+const tokenVer = () => {
```

Comment:
this should be named getAuthorizationHeader, because that is what is being done here